src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsKruxSegments.java

```diff
@@ -0,0 +1,87 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import static com.wikia.webdriver.common.core.Assertion.assertEquals;
+import static com.wikia.webdriver.common.core.Assertion.assertStringContains;
+
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
+import com.wikia.webdriver.common.core.url.Page;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsKruxObject;
+
+import org.testng.annotations.Test;
+
+/**
+ * @ownership AdEngineering
+ */
+public class TestAdsKruxSegments extends TemplateNoFirstLoad {
+
+  /**
+   * Krux loads after onload event happens.
+   *
+   * Unfortunately ads tend to load very slowly on the first page view and it means onload event
+   * happens really late. It breaks our tests as Krux doesn't load in the specified timeout.
+   *
+   * Unless we visit the pages without ads :-)
+   */
+  static private final String NO_ADS_URL_PARAM = "InstantGlobals.wgSitewideDisableGpt=1&InstantGlobals.wgSitewideDisableLiftium=1";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)